src/openzaak/components/zaken/api/serializers/zaken.py

```diff
@@ -802,3 +803,57 @@ class Meta:
     def create(self, validated_data):
         validated_data["zaak"] = self.context["parent_object"]
         return super().create(validated_data)
+
+
+class ZaakContactMomentSerializer(serializers.HyperlinkedModelSerializer):
+    class Meta:
+        model = ZaakContactMoment
+        fields = ("url", "uuid", "zaak", "contactmoment")
+        extra_kwargs = {
+            "url": {"lookup_field": "uuid"},
+            "uuid": {"read_only": True},
+            "zaak": {"lookup_field": "uuid"},
+            "contactmoment": {
+                "validators": [
+                    ResourceValidator(
+                        "ContactMoment", settings.CMC_API_SPEC, get_auth=get_auth
+                    )
+                ]
+            },
+        }
+
+    def create(self, validated_data):
+        with transaction.atomic():
+            zaakcontactmoment = super().create(validated_data)
+
+        # we now expect to be in autocommit mode, i.e. - the transaction before has been
+        # committed to the database as well. This makes it so that the remote
+        # Contactmomenten API can actually retrieve this ZaakContactmomenten we just
+        # created, to validate that we did indeed create the relation information on our
+        # end.
+
+        # we know that we got valid URLs in the initial data
+        contactmoment_url = self.initial_data["contactmoment"]
+        zaak_url = self.initial_data["zaak"]
+
+        # manual transaction management - contactmomenten API checks that the
+        # ZaakContactMoment exists, so that transaction must be committed.
+        # If it fails in any other way, we need to handle that by rolling back
+        # the ZaakContactMoment creation.
+        try:
+            response = create_remote_objectcontactmoment(contactmoment_url, zaak_url)
+        except Exception as exception:
+            zaakcontactmoment.delete()
+            raise serializers.ValidationError(
+                {
+                    "contactmoment": _(
```

Comment:
Not sure if this should be `contactmoment` or `nonFieldErrors` like it was before

Reply:
`contactmoment` looks consistent with ZIO, so I'm in favour of that if the standard does not prescribe anything